templates/user/notification/notification_div.tmpl

```diff
@@ -39,6 +39,10 @@
 							<div class="notifications-icon tw-ml-2 tw-mr-1 tw-self-start tw-mt-1">
 								{{if .Issue}}
 									{{template "shared/issueicon" .Issue}}
+								{{else if .Release}}
+									{{svg "octicon-tag" 16 "text grey"}}
+								{{else if .Commit}}
+									{{svg "octicon-git-commit" 16 "text grey"}}
```

Comment:
Can we prepare the data from backend and make it completely ready in a dedicated struct, and make tmpl only renders, but not do a lot of `if` checks?

Reply:
11bd0eaad4